### PR TITLE
use one logo for fandom powered by wikia

### PR DESCRIPTION
**Pull request by hakubo**

### CODE REVIEW

File: components/global-navigation.md [hunk 1]
```diff
@@ -6,12 +6,8 @@
 	<div class="wds-global-navigation">
 		<div class="wds-global-navigation__content-bar">
 			<a href="#" class="wds-global-navigation__logo">
-				<svg class="wds-global-navigation__logo-fandom" alt="Fandom">
-					<use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#wds-company-logo-fandom"></use>
-				</svg>
-				<span class="wds-global-navigation__logo-powered-by">powered by</span>
-				<svg class="wds-global-navigation__logo-wikia" alt="Wikia">
-					<use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#wds-company-logo-wikia"></use>
+				<svg class="wds-global-navigation__logo-fandom" alt="Fandom powered by Wikia">
```
Comment (rogatty) [inline components/global-navigation.md:9]: `wds-global-navigation__logo-fandom-powered-by-wikia`? Not sure if it changes anything.

File: components/global-navigation.md [hunk 2]
```diff
@@ -86,13 +82,9 @@
...
 		<div class="wds-global-navigation__content-bar">
 			<a href="#" class="wds-global-navigation__logo">
-				<svg class="wds-global-navigation__logo-fandom" alt="Fandom">
-					<use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#wds-company-logo-fandom"></use>
-				</svg>
-				<span class="wds-global-navigation__logo-powered-by">powered by</span>
-				<svg class="wds-global-navigation__logo-wikia" alt="Wikia">
-					<use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#wds-company-logo-wikia"></use>
-				</svg>
+			<svg class="wds-global-navigation__logo-fandom" alt="Fandom powered by Wikia">
```
Comment (rogatty) [inline components/global-navigation.md:85]: One `\t` is missing

**Comment by rogatty** [comment]
Two minor comments but LGTM

File: bower.json [hunk 3]
```diff
@@ -5,6 +5,6 @@
     "url": "git://github.com/Wikia/design-system-reference-page.git"
   },
   "dependencies": {
-    "design-system": "Wikia/design-system#0.3.1"
+    "design-system": "Wikia/design-system#XW-1926-add-fandom-logo-v2"
```
Comment (kvas-damian) [inline bower.json:8]: Probably we shouldn't merge this line

**Comment by kvas-damian** [comment]
Would be great to see only this logo change on vrt. Maybe we should consider merging it to dev.

**Comment by tomasznapieralski** [comment]
LGTM
@kvas-damian we will update VRT on XW-1690

**Pull request merged**